### PR TITLE
Add parameters

### PR DESCRIPTION
**Pull request by charleskawczynski**
- Adds Mean surface temperature
 - Adds Mean change in virtual temperature

### CODE REVIEW

File: src/Planet.jl [hunk 1]
```diff
@@ -127,5 +127,9 @@ function orbit_semimaj end
 function TSI end
 """ Mean sea level pressure (Pa) """
 function MSLP end
+""" Mean surface temperature (K) """
```
Comment (tapios) [inline src/Planet.jl:130]:
```suggestion
""" Mean surface temperature (K) in reference state"""
```

File: src/Planet.jl [hunk 2]
```diff
@@ -127,5 +127,9 @@ function orbit_semimaj end
...
+function T_surf_ref end
+""" Mean change in virtual temperature (K) """
```
Comment (tapios) [inline src/Planet.jl:132]:
```suggestion
""" Mean vertical virtual temperature drop (K) in reference state"""
```

**Comment by tapios** [comment]
This looks fine. But why not changing the drivers correspondingly in the same go?

**Comment by charleskawczynski** [comment]
> This looks fine. But why not changing the drivers correspondingly in the same go?

This must merge here before it can be used in ClimateMachine (CM) because CLIMAParameters is in CM’s deps, which must be updated to use the latest version. If these parameters were defined in CM, we could do it in one go, but then we’d never be able to split out CM into separate packages

**Pull request merged**